content/docs/serverless/serverless-driver.md

```diff
@@ -30,7 +34,7 @@ DATABASE_URL=postgres://<user>:<password>@<endpoint>.<region>.aws.neon.tech/<dbn
 
 ## How to use the driver
 
-To use the Neon serverless driver, you must use the driver's `neon` function. You can use raw SQL queries or [Drizzle-ORM](https://orm.drizzle.team/docs/installation-and-db-connection/postgresql/neon) for type safety. For example: -->
+To use the Neon serverless driver over HTTP, you must use the driver's `neon` function. You can use raw SQL queries or [Drizzle-ORM](https://orm.drizzle.team/docs/installation-and-db-connection/postgresql/neon) for type safety.
```

Comment:
Obviously there are lots of other things that can be used for type safety, such as Kysely or Zapatos. I take it we specifically want to promote Drizzle here?

Reply:
I think that was the idea behind adding the Drizzle example, but I don't think the intention is to exclude others. 
I added mention of Kysley and Zapatos.

Reply:
@raoufchebri Should we add a few other examples?